lib/uiautomator2.js

```diff
@@ -97,6 +97,38 @@ class UiAutomator2Server {
     return resultInfo;
   }
 
+  /**
+   * @typedef {Object} PackageInfo
+   * @property {string} installState
+   * @property {string} appPath
+   * @property {string} appId
+   */
+
+  /**
+   * If the given package information includes servers should uninstall befoer installing new UIA2 servers.
```

Comment:
Checks if server components must be installed from the device under test in scope of the current driver session.

...
returns {boolean} true if any of components is already installed and the other is not installed or the installed one has a newer version